components/service-operator/apis/storage/v1beta1/image_repository_types.go

```diff
@@ -174,3 +182,43 @@ func (s *ImageRepository) GetStackRoleParameters(roleName string) ([]*cloudforma
 	}
 	return params, nil
 }
+
+func (s *ImageRepository) Empty(client sdk.Client) error {
```

Comment:
Can we get unit tests for the calls to `Empty` using the fake aws client to ensure that we are calling the AWS API as we expect to? (applies to both image_repository_types_test and s3_types_tests